lib/screens/rating_page.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rating_bar/flutter_rating_bar.dart';
+import 'package:project_eureka_flutter/screens/home_page.dart';
+
+class RatingPage extends StatefulWidget {
+  @override
+  _RatingPageState createState() => _RatingPageState();
+}
+
+class _RatingPageState extends State<RatingPage> {
+  double _rating;
+  int _ratingBarMode = 1;
+  IconData _selectedIcon;
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      home: Builder(
+        builder: (context) => Scaffold(
+          backgroundColor: Color(0xff00adb5),
+          body: Center(
+              child: Container(
+            decoration: BoxDecoration(
+                color: Colors.white,
+                borderRadius: BorderRadius.all(Radius.circular(20))),
+            height: MediaQuery.of(context).size.height - 250.0,
+            width: MediaQuery.of(context).size.width - 30.0,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                SizedBox(
+                  height: 40.0,
+                ),
+                Center(
+                  child: Column(
+                    children: [
+                      _profilePicture('assets/images/batman.png'),
+                      _heading('How was your session with', 24.0),
+                      _heading('Tony N?', 24.0),
+                      _ratingBar(_ratingBarMode),
+                    ],
+                  ),
+                ),
+                SizedBox(
+                  height: 20.0,
+                ),
+                _rating != null
+                    ? Text(
+                        'Rating: $_rating',
+                        style: TextStyle(fontWeight: FontWeight.bold),
+                      )
+                    : Container(),
+                SizedBox(
+                  height: 40.0,
+                ),
+                Center(
+                  child: _submitButton('Done!'),
```

Comment:
have the button be the same as `EurekaRoundedButton` , a component that Tony N. made for PE